sdk/js/src/utils/aptos.ts

```diff
@@ -0,0 +1,20 @@
+import { sha3_256 } from "js-sha3";
+import { ChainId, CHAIN_ID_APTOS, hex } from "../utils";
+
+export const deriveWrappedAssetAddress = (
```

Comment:
looking at this function, should it now be called `deriveAssetAddress`? since it now supports non-wrapped assets too